rs/tests/src/orchestrator/utils/subnet_recovery.rs

```diff
@@ -1,450 +0,0 @@
-use std::collections::BTreeMap;
```

Comment:
Did you move this file manually or did you call `git mv ...`? The diff seems to suggest you did the former, and that they are now two completely independent pieces of code. But if you do the latter, then you can preserve the git history for this file.

But it's a minor point so feel free to ignore.

Reply:
I did do `git mv` but afterwards I moved large chunks of code out of this file, resulting in a low similarity index (https://stackoverflow.com/questions/54085112/git-mv-and-changes-and-similarity-index) that's why it's showing as a new file. At least this is my hypothesis...